src/services/user.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from '../components/types/User';
+import usersFromServer from '../api/users';
+
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
```

Comment:
null is redundant as method find will return an array 
```suggestion
  return usersFromServer.find(user => user.id === userId);
```